src/preprocessor.rs

```diff
@@ -63,19 +60,14 @@ pub fn lookup_with_config(config: &PreprocessorConfig) -> Result<Box<dyn Preproc
     match config.name.as_str() {
         "separate" => Ok(Box::new(Separate::from_config(&config.config)?)),
         "base64" => Ok(Box::new(Base64::default())),
-        "gzip" => Ok(Box::new(Gzip::default())),
```

Comment:
same here. How is removing all those shorthands a win?

Reply:
See above